nitrates/data_prep/do_data_setup.py

```diff
@@ -325,8 +325,8 @@ def get_event(args):
         N_evfiles = len(ev_data_table)
         logging.info(str(N_evfiles) + " event files found")
 
-        tstarts = Time(ev_data_table.UTCstart.values.astype(np.str), format="isot")
```

Comment:
explain this change?

Reply:
np.str is no longer a thing in newer versions of numpy, which I initially tested with. str is now correct and does also work on the older version of numpy that echo has in its requirments.txt